lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.spec.ts

```diff
@@ -0,0 +1,215 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TranslateModule } from '@ngx-translate/core';
+import { ContentTestingModule } from '../../../testing/content.testing.module';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { SearchDateRangeAdvanced } from './search-date-range-advanced/search-date-range-advanced';
+import { SearchFilterTabbedComponent } from '../search-filter-tabbed/search-filter-tabbed.component';
+import { SearchDateRangeAdvancedComponent } from './search-date-range-advanced/search-date-range-advanced.component';
+import { SearchDateRangeAdvancedTabbedComponent } from './search-date-range-advanced-tabbed.component';
+import { DateRangeType } from "./search-date-range-advanced/date-range-type";
+import { InLastDateType } from "./search-date-range-advanced/in-last-date-type";
```

Comment:
should be ' instead of "

Reply:
These has been resolved in [this](https://github.com/Alfresco/alfresco-ng2-components/pull/8714/commits/bdaa0e52fd3a31ba7c3802c6cdc6a27a733d6057) commit